### PR TITLE
Test v2 data bridge

### PR DESCRIPTION
**Pull request by schwitzguebel**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: docker-compose/swe-idcc-idcf/config/swe-idcc-idcf-data-bridge-application.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  sources:
+    ftp:
+      active: true
+      host: ftp-server
+      port: 21
+      username: gridcapa
+      password: gridcapa
+      base-directory: /data/gridcapa/swe/idcf
+      polling-delay-in-ms: 5000
+      max-messages-per-poll: 10
```
Comment (danielthirion) [inline docker-compose/swe-idcc-idcf/config/swe-idcc-idcf-data-bridge-application.yml:113]: I think "max-pool-size: 2" is missing here

Comment (schwitzguebel) [reply docker-compose/swe-idcc-idcf/config/swe-idcc-idcf-data-bridge-application.yml:113]: thanks I missed that one

File: k8s/bases/cse-valid-d2cc/data-bridge/cse-valid-d2cc-data-bridge-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            time-validity: hourly
+            remote-file-regex: ".*CCC-GSKSel.*|(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_(CO|CA)_GSK_CSE[0-9]{1,2}.xml"
+            source-directory: ""
+            sink-directory: "/GLSKs"
+            file-list-persistence-file: /data/gridcapa/cse-valid-d2cc/ftp-metadata-store.glsk.properties
+          - file-type: TTC_ADJUSTMENT
+            file-regex: "TTC_Adjustment_(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_2D[1-7]_Final_CSE[0-9]{1,2}.xml"
+            time-validity: daily
+            remote-file-regex: ".*CCC-TTCAdj-Final.*|TTC_Adjustment_(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_2D[1-7]_Final_CSE[0-9]{1,2}.xml"
+            source-directory: ""
```
Comment (vbochetRTE) [inline k8s/bases/cse-valid-d2cc/data-bridge/cse-valid-d2cc-data-bridge-configmap.yaml:49]: Source directory should be either "ecp" or "cia" depending on the filetype, but it cannot be identical for all filetypes.

File: k8s/bases/cse-valid-d2cc/kustomization.yaml [hunk 3]
```diff
@@ -2,21 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
-  - data-bridge/cse-valid-d2cc-crac-export-corner-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-d2cc-crac-export-corner-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-d2cc-crac-export-corner-data-bridge-service.yaml
-  - data-bridge/cse-valid-d2cc-crac-import-corner-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-d2cc-crac-import-corner-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-d2cc-crac-import-corner-data-bridge-service.yaml
-  - data-bridge/cse-valid-d2cc-ttc-adjustment-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-d2cc-ttc-adjustment-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-d2cc-ttc-adjustment-data-bridge-service.yaml
+  - data-bridge/cse-valid-d2cc-data-bridge-configmap.yaml
```
Comment (vbochetRTE) [inline k8s/bases/cse-valid-d2cc/kustomization.yaml:5]: Why did you not move the files to the parent folder, as you did for other processes?

File: k8s/bases/cse-valid-idcc/kustomization.yaml [hunk 4]
```diff
@@ -2,21 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
-  - data-bridge/cse-valid-idcc-crac-export-corner-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-crac-export-corner-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-crac-export-corner-data-bridge-service.yaml
-  - data-bridge/cse-valid-idcc-crac-import-corner-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-crac-import-corner-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-crac-import-corner-data-bridge-service.yaml
-  - data-bridge/cse-valid-idcc-ttc-adjustment-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-ttc-adjustment-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-ttc-adjustment-data-bridge-service.yaml
+  - data-bridge/cse-valid-idcc-data-bridge-configmap.yaml
```
Comment (vbochetRTE) [inline k8s/bases/cse-valid-idcc/kustomization.yaml:5]: Why did you not move the files to the parent folder, as you did for other processes?

**Review by kahyami — APPROVED** [review]

**Pull request merged**